src/supertux/menu/editor_converters_menu.cpp

```diff
@@ -62,15 +62,17 @@ EditorConvertersMenu::EditorConvertersMenu() :
   add_hl();
 
   add_file(_("Select Tile Conversion File"), &m_tile_conversion_file, { "sttc" }, "images/converters", false,
-           [this](MenuItem& item) {
-             auto it = m_converters.find(item.get_text());
-             if (it == m_converters.end())
-               return;
-
-             item.set_text("\"" + it->second.title + "\"");
-             item.set_help(it->second.description + (it->second.author.empty() ? "" :
-                           "\n\n" + fmt::format(fmt::runtime(_("By: {}")), it->second.author)));
-           });
+           [this](MenuItem& item, const std::string& file_path, bool in_basedir) {
+            if (in_basedir) {
+              std::string basename = FileSystem::basename(file_path);
+              auto it = m_converters.find(basename);
+              if (it == m_converters.end())
+                return;
+
+              item.set_text("\"" + it->second.title + "\"");
+              item.set_help(it->second.description + (it->second.author.empty() ? "" :
+                            "\n\n" + fmt::format(fmt::runtime(_("By: {}")), it->second.author)));
+            }});
```

Comment:
Needs one space more in tabulation so it's 2 spaces per code style:

```suggestion
             if (in_basedir) {
               std::string basename = FileSystem::basename(file_path);
               auto it = m_converters.find(basename);
               if (it == m_converters.end())
                 return;

               item.set_text("\"" + it->second.title + "\"");
               item.set_help(it->second.description + (it->second.author.empty() ? "" :
                            "\n\n" + fmt::format(fmt::runtime(_("By: {}")), it->second.author)));
             }});
```